src/language-js/printer-estree.js

```diff
@@ -436,6 +436,9 @@ function printPathNoParens(path, options, print, args) {
       const shouldIndentIfInlining =
         parent.type === "AssignmentExpression" ||
         parent.type === "VariableDeclarator" ||
+        parent.type === "ClassProperty" ||
+        parent.type === "TSAbstractClassProperty" ||
+        parent.type === "ClassPrivateProperty" ||
```

Comment:
Does this cover public properties too?

Reply:
Is that you mean?

**Prettier pr-4085**
[Playground link](https://deploy-preview-4085--prettier.netlify.com/playground/#N4Igxg9gdgLgprEAucAbAhgZ0wAgII7AA6UOOADgK4BGqAlmDgGYQTXoBOOAvDiWWQD0gyAFtRCGPwEBGHAGppQkRHGSlOAEwBuEgF8QAGhARyMOtEzJQnDhADuABU4IrKdADcIdACZGQWDDITOiomHDG1BzoYADWcDAAyuQxdFAA5sgwHJQRID4QYMGh4cZp4RwwjtHpoujFYXkAVpgAHgBC0XEJiegSADJpcA2lJpQwVDCaI3kpHBXIIDAAnuRwmGAcdGb+5FuwAOq+MAAWyAAcAAzGexDhB9Hki3vrcBwew8YccACOlHTfaroWr1JAhRrGcKiOhZHJ5TBpdKoOAARUoEHgM2MMHQ1COPlOyE02OidHoGQAwqo6osoNBPiBKOEACq4tzg8J6PRAA)
```sh
--parser typescript
```

**Input:**
```tsx
class A {
  public foobar = 
    //comment
    1 +
    //comment
    2;
}
```

**Output:**
```tsx
class A {
  public foobar =
    //comment
    1 +
    //comment
    2;
}

```

Reply:
yes. that looks good already!